release_tester/full_download_test.py

```diff
@@ -75,6 +76,29 @@ def package_test(dl_opts: DownloadOptions, new_version, new_dlstage, git_version
 
         results.append(test_driver.run_test("all", "all", [dl_new.cfg.version], props))
 
+        enterprise_packages_are_present = "EE" in editions or "EP" in editions
```

Comment:
DRY - I think we will need most this code in all 3 entry points?

Reply:
I'm planning to address this in the next PR. Let's keep this as a temporary measure.